source/developer/mobile-developer-setup.rst

```diff
@@ -855,3 +855,23 @@ Error message
 
 Solution
   The development version of the Mattermost app cannot be installed alongside a release version. Open ``android/app/build.gradle`` and change the applicationId from ``"com.mattermost.react.native"`` to a unique string for your app.
+
+Errors When Running 'make run-ios'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+I just got a new machine and downloaded XCode. I used git to clone the React Native Mattermost project from Github. I have everything ready to go. I run ``make run-ios`` and see
+
+Error message
+  .. code-block:: bash
+
+    xcrun: error: unable to find utility "instruments", not a developer tool or in PATH
+
+Solution
+  The fix for this is pretty simple but not at all intuitive.
```

Comment:
Similarly, this line isn't needed if we're trying to keep the solution short

Reply:
and this